test/ts-types/main.ts

```diff
@@ -0,0 +1,27 @@
+import {getDocument} from "pdfjs-dist";
+
+class MainTest {
+    task: ReturnType<typeof getDocument> | undefined
+
+    constructor(public file: string) {
+    }
+
+    loadPdf() {
+        this.task = getDocument("file://" + this.file);
+        return this.task.promise;
+    }
+
+    get pdf() {
```

Comment:
This method and the one below is unused in the test, so let's remove it to have a minimal test.